### PR TITLE
Changed strange sounding sentence

### PR DESCRIPTION
**Pull request by Brianmanden**
Checkboxes and radios use are built to support HTML-based form validation and provide concise, accessible labels. 

=>

Checkboxes and radio buttons are used to build and support HTML-based form validation and provide concise, accessible labels.

### CODE REVIEW

**Review by midzer — APPROVED** [review]
Sounds good to me :+1:

File: site/docs/4.3/components/forms.md [hunk 1]
```diff
@@ -176,7 +176,7 @@ Default checkboxes and radios are improved upon with the help of `.form-check`,
 
 Disabled checkboxes and radios are supported. The `disabled` attribute will apply a lighter color to help indicate the input's state.
 
-Checkboxes and radios use are built to support HTML-based form validation and provide concise, accessible labels. As such, our `<input>`s and `<label>`s are sibling elements as opposed to an `<input>` within a `<label>`. This is slightly more verbose as you must specify `id` and `for` attributes to relate the `<input>` and `<label>`.
+Checkboxes and radio buttons are used to build and support HTML-based form validation and provide concise, accessible labels. As such, our `<input>`s and `<label>`s are sibling elements as opposed to an `<input>` within a `<label>`. This is slightly more verbose as you must specify `id` and `for` attributes to relate the `<input>` and `<label>`.
```
Comment (patrickhlauke) [inline site/docs/4.3/components/forms.md]: Even shorter: Checkboxes and radio buttons support HTML-based form validation ...

**Review by patrickhlauke — CHANGES_REQUESTED** [review]

File: site/docs/4.3/components/forms.md (hunk 1 above)
Comment (Brianmanden) [reply site/docs/4.3/components/forms.md]: Even better. Let´s go with the short version.

**Comment by patrickhlauke** [comment]
you closed it before it could get merged?

**Comment by Brianmanden** [comment]
Sorry @patrickhlauke .. I am new to contributing to Open Source projects on GitHub.
I´ve created a new pull request.

**Comment by patrickhlauke** [comment]
ah, no worries @Brianmanden :+1:

**Pull request closed without merging**